Lib/test/test_pstats.py

```diff
@@ -33,6 +34,41 @@ def test_add(self):
         stats = pstats.Stats(stream=stream)
         stats.add(self.stats, self.stats)
 
+    def test_sort_stats_int(self):
+        valid_args = {-1: 'stdname',
+                      0: 'calls',
+                      1: 'time',
+                      2: 'cumulative'}
+        for arg_int, arg_str in valid_args.items():
+            self.stats.sort_stats(arg_int)
+            self.assertEqual(self.stats.sort_type,
+                             self.stats.sort_arg_dict_default[arg_str][-1])
+
+    def test_sort_stats_string(self):
+        for arg in SortKey:
```

Comment:
Since this is the string test, we should be iterating through all the possible strings (no longer a given with the new `SortKey` enum).  Listing them all is probably the best way to go:

    for arg in ('calls', 'ncalls', 'cumtime', 'cumulative', ... ):